app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +25,8 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".ui.add.AddEditActivity"
+            android:label="Add/Edit a Book"></activity>
```

Comment:
This should be 2 different label. "Add Book" or "Edit Book". The label should change depending on add or edit.

Reply:
imo this is state related: we might need to judge what to show based on whether or not the parent/caller activity passed a Book object to this activity. (or could be a isbn + username intent argument which which viewmodel can grab data for it)